File: FactoryMethodJS/Car.js
export default class Car {
    constructor({compayName, color, tyreCompany}) {
        this.compayName = compayName;
        this.color = color;
        this.tyreCompany = tyreCompany;
    }
    print = () => {
        console.log(this.compayName)
        console.log(this.color)
        console.log(this.tyreCompany)
    }
}

File: Factory Pattern/src/Car.java
public class Car {
    int numberOfTyres;
    String color;
    String company;

    public Car(int numberOfTyres, String color, String company) {
        this.numberOfTyres = numberOfTyres;
        this.color = color;
        this.company = company;
    }

    public void printDetails() {
        System.out.println(this.company + " " + this.color + " " + this.numberOfTyres);
    }
}
